### PR TITLE
Proposed abstract for therapeutics

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

The [_mSystems_ guidelines](https://msystems.asm.org/sites/default/files/mSys-ITA-12-15-2015.pdf) outline a fairly unusual abstract format:

> Research Articles have structured abstracts with two sections: “Abstract” and “Importance.” The Abstract section should be 250 words or fewer and should concisely summarize the basic content of the paper without presenting extensive  experimental details. The Importance section should be 150 words or fewer and should provide a non-technical explanation of why the work was undertaken.

While ours is not a research article, because the journal is creating a new paper format for us (to continually update), I thought it might be safer to include this!

They also mention [this paper](https://iai.asm.org/content/iai/77/10/4177.full.pdf) for context on importance.

### Related issues

Must be merged after #739 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
Current spelling errors should be fixed in #739

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -1,22 +1,126 @@
-## Prophylactics and Therapeutics
+## Identification and Development of Prophylactics and Therapeutics for COVID-19 
 
 ### Abstract
 
+The novel coronavirus SARS-CoV-2 emerged in late 2019 and went on to significantly impact most countries worldwide.
+Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: SARS-CoV-1, a sister species that emerged in 2002, and MERS-CoV, which emerged in 2012.
+Although both of these previous pandemics were controlled fairly quickly, no vaccines or robust therapeutic interventions were identified.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:7]:
```suggestion
Although both of these previous pandemics were rapidly controlled, no vaccines or robust therapeutic interventions were identified.
```

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -1,22 +1,126 @@
...
+However, previous insights into the immune response to coronaviruses, many made in the context of better understanding SARS and MERS, have been beneficial in identifying approaches to the treatment and prophylaxis of COVID-19.
+A number of potential therapeutics were rapidly identified, leading to a large number of clinical trials investigating a variety of possible therapeutic approaches being initiated early on in the pandemic.
+Similarly, previous advances in vaccine development allowed for the rapid design and testing of candidate prophylactics.
+As a result, by the end of 2020, a small number of therapeutics and vaccines have been authorized by regulatory agencies such as the Food and Drugs Administration in the United States, and many more continue to be under investigation.
+Here, we describe a range of approaches to the treatment and prevention of COVID-19 along with their proposed mechanism of action and the current status of clinical investigation into each candidate.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:12]:
```suggestion
Here, we describe a range of approaches for the treatment and prevention of COVID-19 along with their proposed mechanisms of action and the current status of clinical investigation into each candidate.
```

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -1,22 +1,126 @@
...
+The status of these investigations will continue to evolve, and this review will be updated as progress is made.
+
+### Importance 
+
+The COVID-19 pandemic is a rapidly evolving crisis.
+When the worldwide scientific community shifted to focus on the SARS-CoV-2 virus and the disease it causes, a large number of possible pharmaceutical approaches to treatment and prevention were proposed.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:18]:
```suggestion
When the worldwide scientific community shifted its focus on the SARS-CoV-2 virus and the disease it causes, a large number of possible pharmaceutical approaches for treatment and prevention were proposed.
```

**Review by RLordan — APPROVED** [review]
Hey @rando2 the abstract looks great, will go through the rest of manuscript tomorrow

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -1,22 +1,126 @@
...
+A number of potential therapeutics were rapidly identified, leading to a large number of clinical trials investigating a variety of possible therapeutic approaches being initiated early on in the pandemic.
+Similarly, previous advances in vaccine development allowed for the rapid design and testing of candidate prophylactics.
+As a result, by the end of 2020, a small number of therapeutics and vaccines have been authorized by regulatory agencies such as the Food and Drugs Administration in the United States, and many more continue to be under investigation.
+Here, we describe a range of approaches for the treatment and prevention of COVID-19 along with their proposed mechanisms of action and the current status of clinical investigation into each candidate.
+The status of these investigations will continue to evolve, and this review will be updated as progress is made.
+
+### Importance 
+
+The COVID-19 pandemic is a rapidly evolving crisis.
+When the worldwide scientific community shifted its focus on the SARS-CoV-2 virus and the disease it causes, a large number of possible pharmaceutical approaches for treatment and prevention were proposed.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:18]:
```suggestion
When the worldwide scientific community shifted its focus onto the SARS-CoV-2 virus and the disease it causes, a large number of possible pharmaceutical approaches for treatment and prevention were proposed.
```
Building off @RLordan's edit!

**Comment by rando2** [comment]
Thank you so much @RLordan!

**Comment by rando2** [comment]
merge after #739

**Pull request merged**